client/src/index.jsx

```diff
@@ -1,30 +1,74 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Calendar from './components/Calendar.jsx';
-import Time from './components/Time.jsx';
-import People from './components/People.jsx';
+import axios from 'axios';
+import $ from 'jquery';
+import Reservation from './components/Reservation/Reservation.jsx';
+import ReservationTop from './components/Reservation/ReservationTop.jsx';
+import Order from './components/Order/Order.jsx';
+import BusinessInfo from './components/BusinessInfo/BusinessInfo.jsx';
+import Hours from './components/Hours/Hours.jsx';
+
+import style1 from '../style/style.css';
+import style2 from '../style/calendar.css';
+import style3 from '../style/modal.css';
+import style4 from '../style/table.css';
+import style5 from '../style/sticky.css';
```

Comment:
Should rename style1 - style5 to a clearer naming convention.
Something like styleGeneric, styleCalendar, styleModal, etc.

Reply:
changed!